### ISSUE TITLE
`iveg` and `nbins_veg_vert`

### ISSUE THREAD

**Issue by SorooshMani-NOAA** [schism-dev/pyschism#126]
It seems that the latest SCHISM requires param.nml to have `iveg` and `nbins_veg_vert` as input, otherwise solver fails. This requirement should be reflected in PySCHISM so that the setup generated by PySCHISM works without manual `param.nml` modification.

**Comment by josephzhang8** [comment]
Yes, nbin_veg_vert is now mandatory in CORE/ but iveg is optional.
